### PR TITLE
instructor response comments: wrong user type shown in visibility options

### PR DESCRIPTION
**Pull request by franklingu**
#3025

### CODE REVIEW

**Comment by damithc** [comment]
@kai33 for your review please

**Comment by kai33** [comment]
also call registerResponseCommentsEvent() after add a new one.
otherwise regression

File: src/main/webapp/js/feedbackResponseComments.js [hunk 1]
```diff
@@ -43,6 +43,7 @@ var addCommentHandler = function(e) {
 	                    var newCommentRow = addFormRow.prev();
 	                    newCommentRow.find("form[class*='responseCommentEditForm'] > div > a[id*='button_save_comment_for_edit']").click(editCommentHandler);
 	                    newCommentRow.find("form[class*='responseCommentDeleteForm'] > a").click(deleteCommentHandler);
+	                    registerCheckboxEventForVisibilityOptions();
```
Comment (franklingu) [inline src/main/webapp/js/feedbackResponseComments.js]: @kai33 you are not referring to this?

Comment (kai33) [inline src/main/webapp/js/feedbackResponseComments.js]: this register of event is for student comment.
try the regression yourself, then u will know it

Comment (franklingu) [inline src/main/webapp/js/feedbackResponseComments.js]: yes. you are right. but registerResponseCommentsEvent is just another source of bug

```
    $("form[class*='responseCommentAddForm'] > div > a").click(addCommentHandler);
    $("form[class*='responseCommentEditForm'] > div > a").click(editCommentHandler);
    $("form[class*='responseCommentDeleteForm'] > a").click(deleteCommentHandler);
```

**Comment by kai33** [comment]
thank u!

**Pull request merged**